contracts/exchange-forwarder/contracts/src/interfaces/IForwarderCore.sol

```diff
@@ -25,56 +25,41 @@ import "@0x/contracts-exchange-libs/contracts/src/LibFillResults.sol";
 
 contract IForwarderCore {
 
-    /// @dev Purchases as much of orders' makerAssets as possible by selling up to 95% of transaction's ETH value.
-    ///      Any ZRX required to pay fees for primary orders will automatically be purchased by this contract.
-    ///      5% of ETH value is reserved for paying fees to order feeRecipients (in ZRX) and forwarding contract feeRecipient (in ETH).
-    ///      Any ETH not spent will be refunded to sender.
+    /// @dev Purchases as much of orders' makerAssets as possible by selling as much of the ETH value sent
+    ///      as possible, accounting for order and forwarder fees.
     /// @param orders Array of order specifications used containing desired makerAsset and WETH as takerAsset.
     /// @param signatures Proofs that orders have been created by makers.
-    /// @param feeOrders Array of order specifications containing ZRX as makerAsset and WETH as takerAsset. Used to purchase ZRX for primary order fees.
-    /// @param feeSignatures Proofs that feeOrders have been created by makers.
     /// @param feePercentage Percentage of WETH sold that will payed as fee to forwarding contract feeRecipient.
     /// @param feeRecipient Address that will receive ETH when orders are filled.
     /// @return Amounts filled and fees paid by maker and taker for both sets of orders.
     function marketSellOrdersWithEth(
         LibOrder.Order[] memory orders,
         bytes[] memory signatures,
-        LibOrder.Order[] memory feeOrders,
-        bytes[] memory feeSignatures,
         uint256 feePercentage,
         address payable feeRecipient
     )
         public
         payable
-        returns (
-            LibFillResults.FillResults memory orderFillResults,
-            LibFillResults.FillResults memory feeOrderFillResults
-        );
+        returns (LibFillResults.FillResults memory orderFillResults);
```

Comment:
We should think about what is most useful to return here. It might just be `takerAssetAmountSold`, `makerAssetAmountPurchased`, and `affiliateFeePaid`.